examples/isometric-game/README.md

```diff
@@ -1 +1,3 @@
-Example of making an isometric 2D game.
\ No newline at end of file
+An example of a 2D isometric game. If you ever dreamed of creating you own version of Civilization or Warcraft, this example is for you!
+
+Discover what you can accomplish within an isometric world using smooth animations. Add as many blocks as you want to create whatever you can imagine!
```

Comment:
Even without superlatives, making it sounds like it will allow someone to make something like a AAA strategy games when the example is a very early **adventure game** prototype is misleading if not dishonest.

The best feature of this example is the art done by Mickael. The tile positioning doesn't allow to "Add as many blocks as you want" without having headache because the tiles doesn't align on the grid. I have a version that fix that I could share.

What about a description strait to the point like this:
```
This example shows how to build an isometric word and handles character movements.
```

Reply:
Agree that the reference to AAA strategy game is not the best thing here, because it's far from this and the example is very simple, while making isometric games is still fairly complex.

Thanks for challenging this!

Reply:
I admit I've been a bit cavalier with that!
I'm always amazed by what the extension the community manages to create, I wouldn't be surprised if someone create an extension that allows you to create civilization-like games easily.